release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -3542,4 +3542,29 @@ $ oc adm release info 4.13.24 --pullspecs
 
 [id="ocp-4-13-24-updating"]
 ==== Updating
-To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
\ No newline at end of file
+To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-13-25"]
+=== RHSA-2023:7604 - {product-title} 4.13.25 bug fix and security update
+
+Issued: 2023-12-06
+
+{product-title} release 4.13.25, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:7604[RHSA-2023:7604] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:7606[RHSA-2023:7606] advisory.
```

Comment:
Note matches the following 4.14 release note:

[Bug fixes(https://docs.openshift.com/container-platform/4.14/release_notes/ocp-4-14-release-notes.html#ocp-4-14-2-bug-fixes)